pkg/server/datastore/datastore.go

```diff
@@ -40,6 +40,10 @@ type DataStore interface {
 	PruneRegistrationEntries(ctx context.Context, expiresBefore time.Time) error
 	UpdateRegistrationEntry(context.Context, *common.RegistrationEntry, *common.RegistrationEntryMask) (*common.RegistrationEntry, error)
 
+	// Entries Events
+	ListRegistrationEntriesEvents(context.Context, *ListRegistrationEntriesEventsRequest) (*ListRegistrationEntriesEventsResponse, error)
+	PruneRegistrationEntriesEvents(ctx context.Context, olderThan time.Duration) error
```

Comment:
for consistency, I think we may add var names to both or remove to both

Reply:
Added the var names. That file is inconsistent overall with some have var names and some not.